2023-06-18 15:38:04,184        INFO         server.py                 Запущен сервер, порт для подключений: 8079, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 15:42:53,329        INFO         server.py                 Запущен сервер, порт для подключений: 8079, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 15:42:53,371        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50198)
2023-06-18 15:42:53,376        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50199)
2023-06-18 15:42:53,377        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50200)
2023-06-18 15:42:53,905        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,905        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687092173.3716154, 'user': {'account_name': 'test_1'}}
2023-06-18 15:42:53,907        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 15:42:53,909        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 1687092173.3716154, 'user': {'account_name': 'test_1'}}, [], <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, [<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>], {'test_1': <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,910        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,910        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687092173.3726137, 'user': {'account_name': 'test_2'}}
2023-06-18 15:42:53,912        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 15:42:53,913        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 1687092173.3726137, 'user': {'account_name': 'test_2'}}, [], <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, [<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>], {'test_1': <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, 'test_2': <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,914        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:42:53,914        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687092173.3775995, 'user': {'account_name': 'test_3'}}
2023-06-18 15:42:53,915        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 15:42:53,916        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 1687092173.3775995, 'user': {'account_name': 'test_3'}}, [], <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>, [<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>], {'test_1': <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, 'test_2': <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, 'test_3': <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:43:36,699        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:43:36,699        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test_3', 'to': 'test_2', 'time': 1687092216.5385835, 'mess_text': 'Hello'}
2023-06-18 15:43:36,700        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'message', 'from': 'test_3', 'to': 'test_2', 'time': 1687092216.5385835, 'mess_text': 'Hello'}, [{'action': 'message', 'from': 'test_3', 'to': 'test_2', 'time': 1687092216.5385835, 'mess_text': 'Hello'}], <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>, [<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>], {'test_1': <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, 'test_2': <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, 'test_3': <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:43:36,702        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, {'action': 'message', 'from': 'test_3', 'to': 'test_2', 'time': 1687092216.5385835, 'mess_text': 'Hello'}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_message
2023-06-18 15:43:36,702        INFO         server.py                 Отправлено сообщение пользователю test_2 от пользователя test_3.
2023-06-18 15:43:36,703        DEBUG        server.py                 Была вызвана функция process_message c параметрами ({'action': 'message', 'from': 'test_3', 'to': 'test_2', 'time': 1687092216.5385835, 'mess_text': 'Hello'}, {'test_1': <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, 'test_2': <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, 'test_3': <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>}, [<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, <socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>, <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>]), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:44:16,783        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=432, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50199)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:44:16,783        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687092256.3393123, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}
2023-06-18 15:44:16,785        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'exit', 'time': 1687092256.3393123, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}, [], <socket.socket [closed] fd=-1, family=2, type=1, proto=0>, [<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>], {'test_1': <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>, 'test_3': <socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:44:23,921        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=316, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50200)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:44:23,921        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687092263.4512267, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}
2023-06-18 15:44:23,922        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'exit', 'time': 1687092263.4512267, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}, [], <socket.socket [closed] fd=-1, family=2, type=1, proto=0>, [<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>], {'test_1': <socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 15:44:30,039        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=428, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50198)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 15:44:30,039        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687092269.6031377, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}
2023-06-18 15:44:30,040        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'exit', 'time': 1687092269.6031377, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}, [], <socket.socket [closed] fd=-1, family=2, type=1, proto=0>, [], {}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:36,938        INFO         server.py                 Запущен сервер, порт для подключений: 8079, адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 16:24:37,407        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50418)
2023-06-18 16:24:37,424        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50419)
2023-06-18 16:24:37,438        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 50420)
2023-06-18 16:24:37,456        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,456        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687094677.4079187, 'user': {'account_name': 'test_3'}}
2023-06-18 16:24:37,458        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 16:24:37,459        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 1687094677.4079187, 'user': {'account_name': 'test_3'}}, [], <socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, [<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>], {'test_3': <socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,967        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,967        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687094677.4240186, 'user': {'account_name': 'test_2'}}
2023-06-18 16:24:37,968        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 16:24:37,969        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 1687094677.4240186, 'user': {'account_name': 'test_2'}}, [], <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, [<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>], {'test_3': <socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, 'test_2': <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,970        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:24:37,970        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687094677.4380956, 'user': {'account_name': 'test_1'}}
2023-06-18 16:24:37,972        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 16:24:37,973        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'presence', 'time': 1687094677.4380956, 'user': {'account_name': 'test_1'}}, [], <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>, [<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>], {'test_3': <socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, 'test_2': <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, 'test_1': <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:22,699        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:22,699        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687094722.5156102, 'mess_text': 'Hello World'}
2023-06-18 16:25:22,700        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687094722.5156102, 'mess_text': 'Hello World'}, [{'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687094722.5156102, 'mess_text': 'Hello World'}], <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>, [<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>], {'test_3': <socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, 'test_2': <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, 'test_1': <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:22,702        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687094722.5156102, 'mess_text': 'Hello World'}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_message
2023-06-18 16:25:22,702        INFO         server.py                 Отправлено сообщение пользователю test_3 от пользователя test_1.
2023-06-18 16:25:22,703        DEBUG        server.py                 Была вызвана функция process_message c параметрами ({'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687094722.5156102, 'mess_text': 'Hello World'}, {'test_3': <socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, 'test_2': <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, 'test_1': <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>}, [<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>, <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>]), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:33,932        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=344, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50418)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:33,932        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687094733.636207, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}
2023-06-18 16:25:33,934        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'exit', 'time': 1687094733.636207, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}, [], <socket.socket [closed] fd=-1, family=2, type=1, proto=0>, [<socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>], {'test_2': <socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>, 'test_1': <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:42,078        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=368, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50419)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:42,078        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687094741.883918, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}
2023-06-18 16:25:42,079        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'exit', 'time': 1687094741.883918, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}, [], <socket.socket [closed] fd=-1, family=2, type=1, proto=0>, [<socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>], {'test_1': <socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:48,735        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=372, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 50420)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции main
2023-06-18 16:25:48,735        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687094748.508022, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}
2023-06-18 16:25:48,736        DEBUG        server.py                 Была вызвана функция process_client_message c параметрами ({'action': 'exit', 'time': 1687094748.508022, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}, [], <socket.socket [closed] fd=-1, family=2, type=1, proto=0>, [], {}), {}. Вызов из модуля __main__. Вызов из функции main().Вызов из функции main
2023-06-18 19:28:55,522        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 19:31:22,260        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 19:31:58,356        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 19:44:07,362        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 19:45:06,825        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 19:45:26,882        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 19:45:26,889        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 51127)
2023-06-18 19:45:26,898        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 51128)
2023-06-18 19:45:26,923        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 51129)
2023-06-18 19:45:26,941        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=440, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51127)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 19:45:26,941        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687106726.8898008, 'user': {'account_name': 'test_1'}}
2023-06-18 19:45:26,943        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=440, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51127)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 19:45:26,947        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51128)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 19:45:26,948        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687106726.897781, 'user': {'account_name': 'test_2'}}
2023-06-18 19:45:26,951        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51128)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 19:45:27,471        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=444, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51129)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 19:45:27,471        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687106726.9237528, 'user': {'account_name': 'test_3'}}
2023-06-18 19:45:27,473        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=444, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51129)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 19:46:08,246        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=440, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51127)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 19:46:08,246        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687106767.830598, 'mess_text': 'My friends'}
2023-06-18 19:46:08,248        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=444, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51129)>, {'action': 'message', 'from': 'test_1', 'to': 'test_3', 'time': 1687106767.830598, 'mess_text': 'My friends'}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_message
2023-06-18 19:46:08,248        INFO         server.py                 Отправлено сообщение пользователю test_3 от пользователя test_1.
2023-06-18 19:46:36,214        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=440, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51127)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 19:46:36,214        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687106796.1666088, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}
2023-06-18 19:46:43,321        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=444, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51129)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 19:46:43,321        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687106802.8793113, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}
2023-06-18 19:46:49,953        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=336, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51128)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 19:46:49,953        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687106809.9426124, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}
2023-06-18 21:06:30,051        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 21:06:30,144        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 51305)
2023-06-18 21:06:30,146        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 51306)
2023-06-18 21:06:30,146        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 51307)
2023-06-18 21:06:30,673        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=452, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51305)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:06:30,673        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687111590.1445096, 'user': {'account_name': 'test_1'}}
2023-06-18 21:06:30,673        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=452, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51305)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 21:06:30,676        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=464, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51306)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:06:30,676        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687111590.145506, 'user': {'account_name': 'test_2'}}
2023-06-18 21:06:30,678        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=464, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51306)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 21:06:30,679        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=468, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51307)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:06:30,679        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687111590.145506, 'user': {'account_name': 'test_3'}}
2023-06-18 21:06:30,681        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=468, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51307)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 21:07:16,446        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=452, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51305)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:07:16,446        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test_1', 'to': 'test_2', 'time': 1687111636.0479417, 'mess_text': 'Hi friends'}
2023-06-18 21:07:16,448        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=464, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51306)>, {'action': 'message', 'from': 'test_1', 'to': 'test_2', 'time': 1687111636.0479417, 'mess_text': 'Hi friends'}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_message
2023-06-18 21:07:16,448        INFO         server.py                 Отправлено сообщение пользователю test_2 от пользователя test_1.
2023-06-18 21:08:01,710        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=464, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51306)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:08:01,710        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test_2', 'to': 'test_3', 'time': 1687111681.3847148, 'mess_text': 'Hello'}
2023-06-18 21:08:01,712        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=468, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51307)>, {'action': 'message', 'from': 'test_2', 'to': 'test_3', 'time': 1687111681.3847148, 'mess_text': 'Hello'}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_message
2023-06-18 21:08:01,712        INFO         server.py                 Отправлено сообщение пользователю test_3 от пользователя test_2.
2023-06-18 21:08:51,563        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=468, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51307)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:08:51,563        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test_3', 'to': 'test_1', 'time': 1687111731.4005957, 'mess_text': 'goodbye'}
2023-06-18 21:08:51,565        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=452, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51305)>, {'action': 'message', 'from': 'test_3', 'to': 'test_1', 'time': 1687111731.4005957, 'mess_text': 'goodbye'}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_message
2023-06-18 21:08:51,565        INFO         server.py                 Отправлено сообщение пользователю test_1 от пользователя test_3.
2023-06-18 21:09:00,174        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=468, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51307)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:09:00,174        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687111739.6808212, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}
2023-06-18 21:09:06,792        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=464, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51306)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:09:06,792        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687111746.703971, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}
2023-06-18 21:09:13,916        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=452, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 51305)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:09:13,916        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687111753.456341, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}
2023-06-18 21:51:26,515        CRITICAL     descriptrs.py             Попытка запуска сервера с указанием неподходящего адреса .
2023-06-18 21:51:48,199        CRITICAL     descriptrs.py             Попытка запуска сервера с указанием неподходящего адреса .
2023-06-18 21:56:08,294        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 21:56:21,851        INFO         server.py                 Запущен сервер, порт для подключений: 8079 , адрес с которого принимаются подключения: . Если адрес не указан, принимаются соединения с любых адресов.
2023-06-18 21:56:21,856        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 52604)
2023-06-18 21:56:21,862        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 52605)
2023-06-18 21:56:21,887        INFO         server.py                 Установлено соединение с ПК ('127.0.0.1', 52606)
2023-06-18 21:56:21,904        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=348, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52604)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:56:21,905        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687114581.8568363, 'user': {'account_name': 'test_1'}}
2023-06-18 21:56:21,906        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=348, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52604)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 21:56:21,908        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52605)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:56:21,908        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687114581.8628204, 'user': {'account_name': 'test_2'}}
2023-06-18 21:56:21,910        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52605)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 21:56:22,420        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=424, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52606)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:56:22,420        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'presence', 'time': 1687114581.8868074, 'user': {'account_name': 'test_3'}}
2023-06-18 21:56:22,421        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=424, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52606)>, {'response': 200}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_client_message
2023-06-18 21:57:18,455        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52605)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:57:18,455        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'message', 'from': 'test_2', 'to': 'test_1', 'time': 1687114638.4094841, 'mess_text': 'goodmorning'}
2023-06-18 21:57:18,457        DEBUG        server.py                 Была вызвана функция send_message c параметрами (<socket.socket fd=348, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52604)>, {'action': 'message', 'from': 'test_2', 'to': 'test_1', 'time': 1687114638.4094841, 'mess_text': 'goodmorning'}), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции process_message
2023-06-18 21:57:18,457        INFO         server.py                 Отправлено сообщение пользователю test_1 от пользователя test_2.
2023-06-18 21:57:24,053        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=420, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52605)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:57:24,053        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687114644.0007727, 'account_name': 'test_2', 'mess_text': 'Пользователь test_2 вышел из чата.'}
2023-06-18 21:57:30,174        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=348, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52604)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:57:30,174        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687114649.7693386, 'account_name': 'test_1', 'mess_text': 'Пользователь test_1 вышел из чата.'}
2023-06-18 21:57:34,762        DEBUG        server.py                 Была вызвана функция get_message c параметрами (<socket.socket fd=424, family=2, type=1, proto=0, laddr=('127.0.0.1', 8079), raddr=('127.0.0.1', 52606)>,), {}. Вызов из модуля utils. Вызов из функции main().Вызов из функции handler_message
2023-06-18 21:57:34,762        DEBUG        server.py                 Получено сообщение от клиента : {'action': 'exit', 'time': 1687114654.6736925, 'account_name': 'test_3', 'mess_text': 'Пользователь test_3 вышел из чата.'}
2023-06-18 21:59:29,507        CRITICAL     descriptrs.py             Попытка запуска сервера с указанием неподходящего адреса 456.14.15.23.
